src/test/e2e/13_zarf_package_generate_test.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package test provides e2e tests for Zarf.
+package test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/pkg/layout"
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestZarfDevGenerate(t *testing.T) {
+	t.Log("E2E: Zarf Dev Generate")
+
+	t.Run("Test arguments and flags", func(t *testing.T) {
+		stdOut, stdErr, err := e2e.Zarf("dev", "generate")
+		require.Error(t, err, stdOut, stdErr)
+
+		stdOut, stdErr, err = e2e.Zarf("dev", "generate", "podinfo")
+		require.Error(t, err, stdOut, stdErr)
+
+		stdOut, stdErr, err = e2e.Zarf("dev", "generate", "podinfo", "--url", "https://zarf.dev")
+		require.Error(t, err, stdOut, stdErr)
+	})
```

Comment:
Assuming you can switch to `MarkFlagRequired` I would feel comfortable deleting this test